pkg/types/baremetal/defaults/platform.go

```diff
@@ -18,6 +18,8 @@ const (
 	HardwareProfile         = "default"
 	APIVIP                  = ""
 	IngressVIP              = ""
+	BootstrapMachineOSImage = ""
+	ClusterMachineOSImage   = ""
```

Comment:
these are not used.

Reply:
I guess they are redundant anyway. Will remove them.

Reply:
I'll just remove it because we don't have a real default value here.